src/plugins/lua/elastic.lua

```diff
@@ -121,29 +121,26 @@ local function get_general_metadata(task)
   local r = {}
   local ip_addr = task:get_ip()
 
-  r.webmail = false
-
   if ip_addr  and ip_addr:is_valid() then
     r.is_local = ip_addr:is_local()
-    local origin = task:get_header('X-Originating-IP')
-    if origin then
-      origin = string.sub(origin, 2, -2)
-      local rspamd_ip = require "rspamd_ip"
-      local test = rspamd_ip.from_string(origin)
-
-      if test and test:is_valid() then
-        r.webmail = true
-        r.ip = origin
-      else
-        r.ip = tostring(ip_addr)
-      end
-    else
-      r.ip = tostring(ip_addr)
-    end
+    r.ip = tostring(ip_addr)
   else
     r.ip = '127.0.0.1'
   end
 
+  r.webmail = false
+  r.sender_ip = 'unknown'
```

Comment:
Why do we need `webmail` and `sender_ip` both?

Reply:
I thought about existing people setups (possibly filters, etc) so decided to leave webmail bool as-is, to not break it for existing setups. Generally, if `sender_ip` isn't unknown then it means that it was sent via webmail, yes.

Reply:
No, I think your arguments are perfectly valid, thank you!